Task02-PixelPerfect/characters.scss

```diff
@@ -0,0 +1,51 @@
+/**
+ * All Rights Reserved
+ * This software is property information of AKUREY S.A.
+ * Contact us at contact@akurey.com
+ * @summary short description for the file
+ */
+
+%character-card-text-properties{
+    font-family: 'Roboto Slab';
+    font-style: normal;
+    font-weight: 700;
+    font-size: 22px;
+    line-height: 28px;
+    color: #FFF;
+}
+
+.grid-container {
+    display: flex;
+    gap: 20px;
+}
+
+.grid-item {
+    position: relative;
+    display: flex; 
+    justify-content: center;
+
+    width: 219.74px;
+    height: 279px;
+}
+
+.text-item{
+    @extend %character-card-text-properties;
+    position: absolute;
+    top: 211.36px;
+    z-index: 15;
+}
+
+.filter-item{
+    width: 100%; 
+    height: 100%; 
+    object-fit: cover;
+    
+    background: linear-gradient(179.8deg, rgba(255, 255, 255, 0) 50.33%, #000000 97.62%);
+    mix-blend-mode: multiply;
+    border-radius: 19.7522px;
+    z-index: 10;
+}
+
+.image-item{
+    position: absolute;
+}
```

Comment:
this scss file has not been used yet, right? 
I'll review it once it's applying changes on the actual page.

Reply:
The scss is being compiled and generating the css file that the html uses

Reply:
Im compiling it with the Live Sass compiler extension for vsc

Reply:
Got it, we're gonna apply the same changes here, with the decimals and the spaces before the '{' brackets.
To these changes are going to be applied on the actual css file

Reply:
Done, I applied those changes on the scss and are being reflected on the css, the spaces before the brackets on the css are placed automatically after compiling the scss